aerolyzer/test_location.py

```diff
@@ -0,0 +1,94 @@
+from retrieve_image_data import RtrvData as Data
+import location
+import image_restriction_main
+
+def test_location_all():
+        filepath = input("Input File Path for Test Image: ")
+        testspassed  = 0
+        if(test_get_coord(filepath)==0):
+                print "test_get_coord passed\n"
+                testspassed+=1
+        else:
+                print "test_get_coord failed\n"
+
+        if(test_coord_to_zip(filepath)==0):
+                print "test_coord_to_zip passed\n"
+                testspassed+=1
+        else:
+                print "test_coord_to_zip failed\n"
+
+        if(test_zip_to_coord(filepath)==0):
+                print "test_zip_to_coord passed\n"
+                testspassed+=1
+        else:
+                print "test_zip_to_coord failed\n"
+
+        if(test_sun_position(filepath)==0):
+                print "test_sun_position passed\n"
+                testspassed+=1
+        else:
+                print "test_sun_position failed\n"
+
+        print "Number of Tests passed "+ str(testspassed) +"/4\n"
+
+
+def test_get_coord(filename):
+        if(image_restriction_main.check_image(filename)):
+                imagedata = Data(filename)
+                exifData = imagedata.get_exif(filename, True, False)
+                coords = location.get_coord(exifData)
+                print "\tOriginal image data: ("+exifData['gps gpslatitude'][1:-1]+","+exifData['gps gpslongitude'][1:-1]+")\n"
+                print "\tConverted image data: ("+str(coords[0])+","+str(coords[1])+")\n"
+                return 0
+                
+        else:
+                print "\tImage failed a restriction test.\n"
+                return 1
+
+def test_coord_to_zip(filename):
+        if(image_restriction_main.check_image(filename)):
+                imagedata = Data(filename)
+                exifData = imagedata.get_exif(filename, True, False)
+                coords = location.get_coord(exifData)
+                zipcode = location.coord_to_zip(coords)
+                print "\tOriginal image data: ("+exifData['gps gpslatitude'][1:-1]+","+exifData['gps gpslongitude'][1:-1]+")\n"
+		print "\tFetched image data: ZIP = "+zipcode+"\n"
+                if(zipcode=="99999"):
```

Comment:
the logic of this function needs fixing. What is it testing? That is what the if/else should be around.